pom.xml

```diff
@@ -122,6 +122,11 @@
         <artifactId>compiler</artifactId>
         <version>${dep.spullara.mustache.compiler.version}</version>
       </dependency>
+      <dependency>
+        <groupId>com.google.code.gson</groupId>
+        <artifactId>gson</artifactId>
+        <version>2.8.6</version>
```

Comment:
```suggestion
        <version>${dep.gson.version}</version>
```

Reply:
Modified. If I took the time to define the dependency version, I guess I should use it, eh?